nipype/pipeline/engine/nodes.py

```diff
@@ -189,12 +188,11 @@ def interface(self):
 
     @property
     def result(self):
-        if self._result:
-            return self._result
-        else:
-            cwd = self.output_dir()
-            result, _, _ = self._load_resultfile(cwd)
-            return result
+        # Cache first
+        if not self._result:
```

Comment:
we don't want to do this i think. this will increase memory consumption. we want results to be loaded on the fly.

Reply:
Okay, I'll see how to improve the process

Reply:
Done, no `self._result` anymore.